LayoutTests/imported/w3c/web-platform-tests/content-security-policy/script-src/hash-always-converted-to-utf-8/iso-8859-1.html

```diff
@@ -10,9 +10,9 @@
       window.addEventListener("securitypolicyviolation", t1.unreached_func("Should not have fired a spv"));
     </script>
 
-    <!-- � (micro sign) has the value of 0xB5 in latin-1 and of 0xC2B5 in utf-8 but the hash value should be the same as the utf-8 computed one -->
+    <!-- � (micro sign) has the value of 0xB5 in latin-1 and of 0xC2B5 in utf-8 but the hash value should be the same as the utf-8 computed one -->
```

Comment:
style checker is failing due to these micro signs (lines 13 and 15). They're encoded latin-1 but the checker is assuming UTF-8 which is causing it to fail to decode.

Reply:
https://bugs.webkit.org/show_bug.cgi?id=245690

Reply:
Interesting!